docs/types.rst

```diff
@@ -301,153 +273,141 @@ Where ``x`` is a byte array and ``_start`` as well as ``_len`` are integer value
 .. index:: !bytes
 
 Fixed-size Byte Arrays
-======================
+----------------------
 
 **Keyword:** ``Bytes``
 
 A byte array with a fixed size.
```

Comment:
`Bytes` is a dynamic type

Reply:
I think the idea here is that because the size in memory is fixed, the type itself is "fixed size".

Reply:
"Fixed" strikes me as another way to say "static", which they're definitely not static.

Reply:
How about just "Byte Arrays"?